views/showcase.html

```diff
@@ -76,7 +76,7 @@ <h3 class="text-xl font-semibold">
           {{ resource.descriptionHtml | safe }}
         </p>
         <nav>
-          <a class="data-file-type mr-1" href="{{ resource.downloadPath }}">
+          <a class="data-file-type mr-1" href="{{ resource.path }}">
```

Comment:
I think ODDK is using frontend-v2@v1.0.0 which provides `downloadPath`

Reply:
You are right. This was long time ago, but if I remember well, I tested this in other browsers and noticed that download links not forming well in IE with `downloadPath` parameter. Anyway, I revert this changes as you suggested.